hilti/runtime/include/types/tuple.h

```diff
@@ -79,11 +79,12 @@ constexpr auto has_value(const Tuple<Ts...>& t) {
  * @param t the tuple
  */
 template<size_t Idx, typename... Ts>
-constexpr auto& get(const Tuple<Ts...>& t) {
-    if ( const auto& x = std::get<Idx>(t); x.has_value() )
-        return *x;
-    else
+const auto& get(const Tuple<Ts...>& t) {
```

Comment:
I do not see this or the change below make a measurable difference on my machine (even inside noise only tiny movement).